pkg/reporter/klog.go

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This seems unrelated to the globalnet change and should be part of a separate PR.

Reply:
yeah I included these so the add-on has some choice of implementations to use instead of spinning it's own when the globanet APIs are used. But I'll move to another PR.